Detections/W3CIISLog/RareClientFileAccess.yaml

```diff
@@ -0,0 +1,31 @@
+id: a787a819-40df-4c9f-a5ae-850d5a2a0cf6
+name: URI requests from single client
+description: |
+  'This will look for connections to files on the server that are requested by only a single client. This analytic will be effective where an actor is utilising relatively static operational IP addresses. The threshold can be modified. The larger the execution window for this query the more reliable the results returned.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+query: |
+
+let timeRange = 7d;
```

Comment:
This is failing as there needs to be 2 spaces before each line of the query itself.  Please fix.

Reply:
Thanks shain, fixed this now.